server/src/utils/fight/fightMethods.ts

```diff
@@ -1939,9 +2046,10 @@ export const playFighterTurn = (
         ? randomBetween(0, 1) === 0
           ? 'thrown'
           : 'melee'
-        // 1/33 chance to throw a weapon otherwise
+        // 1/33 chance to throw a weapon otherwise if no deflect on weapon
```

Comment:
I really don't like this change, it's too complex of a rule for users

Reply:
I agree, it's a cheap solution that will save some bug reports though...

Reply:
Alright I found a better solution